lib/client.js

```diff
@@ -182,6 +182,46 @@ Client.prototype._defineMethod = function(method, location) {
   };
 };
 
+Client.prototype._isSequenceRequired = function(method) {
+  try {
+    var tns = this.wsdl.definitions.xmlns.tns;
+    var childrenName = this.wsdl.definitions.schemas[tns].complexTypes.pullFileParams.children[0].name;
```

Comment:
are these `pullFileParams` specific to your project or are they defined in the specification?